modules/dnn/src/torch/torch_importer.cpp

```diff
@@ -867,12 +867,12 @@ struct TorchImporter
             }
             else if (nnName == "SoftMax")
             {
-                newModule->apiType = "SoftMax";
+                newModule->apiType = "Softmax";
                 curModule->modules.push_back(newModule);
             }
             else if (nnName == "LogSoftMax")
             {
-                newModule->apiType = "SoftMax";
+                newModule->apiType = "Softmax";
                 layerParams.set("log_softmax", true);
                 curModule->modules.push_back(newModule);
             }
```

Comment:
Maybe combine `SoftMax` with `LogSoftMax` to avoid code duplication?

Reply:
done